tests/ft_calloc_test.cpp

```diff
@@ -21,6 +22,28 @@ int main(void)
 	/* 1 */ check(!memcmp(p, e, 4));
 	/* 2 */ mcheck(p, 4); free(p); showLeaks();
 	/* 3 */ check(ft_calloc(SIZE_MAX, SIZE_MAX) == NULL);
+
+	/* @evportel */
```

Comment:
Since the following these behaviors are not specified by the documentation, maybe we could add a message before them.
Something like: Following tests are not backed by function documentation.
What do you think?

Reply:
Okay, it's really good to leave a message letting me know